cmd/tether/attach.go

```diff
@@ -467,21 +487,32 @@ func (t *attachServerSSH) globalMux(reqchan <-chan *ssh.Request) {
 	// ContainersReq will close this channel
 	t.askedAndAnswered = make(chan struct{})
 
+	var pendingFn func()
 	for req := range reqchan {
-		var pendingFn func()
-		var payload []byte
+		payload := []byte{}
```

Comment:
this causes allocation of a byte array pointer using ```var payload []byte``` is actually better especially if there is a chance that no data will be added.